estate/models/estate_property.py

```diff
@@ -0,0 +1,76 @@
+from odoo import models, fields
+
+class EstateProperty(models.Model):
```

Comment:
```suggestion


class EstateProperty(models.Model):
```
we normally add 2 empty lines before defining class
:) refer codebase

Reply:
Noted